project/assets/login.less

```diff
@@ -0,0 +1,117 @@
+body {
+    background-color: rgb(82, 86, 89);
+    margin-top: 50px;
+    margin-left: 0;
+}
+
+/* First section, including the 'feedback' background image and the login form */
+
+.login-form {
+    opacity: 0.8;
+}
+
+.login-form-background-image {
+    width: 1897px;
+    opacity: 0.8;
+}
+
+.login-form-box {
+    align-self: center;
+    line-height: 61px;
+    width: 400px;
+    height: 369px;
+    position: absolute;
+    align-items: center;
+    left: 25%;
+    right: 25%;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.5), 0 6px 20px 0 rgba(0, 0, 0, 0.5);
+}
+
+.login-form-box-skin {
+    background-color: rgb(255, 255, 255);
+}
+
+.login-form-box-header {
+    font-size: 25px;
+    height: 60px;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.5), 0 6px 20px 0 rgba(0, 0, 0, 0.5);
+}
+
+.login-form-box-header-skin {
+    background-color: #03a9f4;
+}
+
+.login-form-box-content {
+    width: 100%;
+}
+
+.button {
+    border: none;
+    width: 120px;
+    height: 40px;
+    text-align: center;
+    text-decoration: none;
+    font-size: 16px;
+    border-radius: 2px;
```

Comment:
border-radius and box shadow are part of the skin. Also most probably some of these props combination are duplicated in other placed and should be extracted into a standalone class( probably font-size together with text-decoration and/or text-align ? )